### ISSUE TITLE
[Package Request]: Reiner SCT

### ISSUE THREAD

**Issue by Karl-WE** [microsoft/winget-pkgs#113811]
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://help.reiner-sct.com/de/support/solutions/articles/101000475765
Publisher: REINER Kartengeräte GmbH und Co. KG
Package Name: ReinerSCT
Description: ReinerSCT is a driver and tool package for a range of popular card reader products from Reiner, If you are using a previous version of the previous Installshield based application, use winget to remove it first. A direct upgrade is not possible.
Package Version: 1.2.0
Installer URL: https://dp.reiner-sct.com/downloads/dp/dp_1_2_0.exe

**Comment by Eddga** [comment]
I submitted a PR for it. I hope you're ok with that I went with the latest version instead of 1.2.0 (which probably just was the latest version by the time you raised this request).
The program supports multiple languages like German and Slovakian and I even went so far to create separate locale manifests for that. But in the end I wasn't able to successfully build a installer manifest file with the individual languages. So I dropped that part and went with simple English manifest files.
